Базовая работа с кодом

LeoBloom228 opened the pull request:
Выполнение ДЗ 5 завершено.

LeoBloom228 committed:
Задание 1 в рамках ДЗ 5.

LeoBloom228 committed:
Задание 2 в рамках ДЗ 5.

LeoBloom228 committed:
Задание 3 в рамках ДЗ 5

LeoBloom228 committed:
Задание 4 в рамках ДЗ 5.
Значения a и b вводим с клавиатуры: 0 и 0.99 соответственно.

LeoBloom228 committed:
Задание 4 в рамках ДЗ 5.
Перевел зацитированные пояснения в рамках данного задания на английский

tak333 commented on Homework-5/file1.cpp:
```diff
@@ -0,0 +1,145 @@
+//#include <cmath>
+//#include <iomanip>
+
+//int main()
```
У вас не должно быть закомментированного кода.
Все задания должны быть вынесены в функции, отличные от main.
Как и в предыдущих д/з.

LeoBloom228 replied:
Убрал цитирование у кода. Сделал вспомогательные функции.

tak333 commented on Homework-5/file1.cpp:
```diff
@@ -0,0 +1,145 @@
+
+    //for (int n = 1; n <= k; ++n) 
+    //{
+        //sum += std::pow(x, 2 * (n - 1)) / std::pow(2 + 4 * (n - 1), 2);
```
Где возможно необходимо использовать рекуррентные отношения.

tak333 commented on Homework-5/file1.cpp:
```diff
@@ -0,0 +1,145 @@
+// Auxiliary function for calculating a member of a series
+double term(int n, double x) 
+{
+    return std::pow(x, 2 * n + 1) / ((2 * n + 1) * std::pow(2, n));
```
Нужно использовать рекуррентные отношения: то есть рассчитывать значение выражения на базе предыдущего.

LeoBloom228 replied:
Здесь, в задании 4 убрал вспомогательную функцию term и в функции f сделал рекуррентное вычисление с помощью current_term

LeoBloom228 committed:
Добавил вспомогательные функции. Убрал цитирование у кода.
в задании 2 добавил переменную numerator, которая накапливает значения х рекурсивно.
В задании 4 убрал вспомогательную функцию term и в функции f сделал рекуррентное вычисление с помощью current_term
